src/DocBlock/Annotation.php

```diff
@@ -178,7 +178,7 @@ public function getTypeExpression(): ?TypeExpression
     public function getVariableName()
     {
         $type = preg_quote($this->getTypesContent() ?? '', '/');
-        $regex = "/@{$this->tag->getName()}\\s+({$type}\\s*)?(&\\s*)?(\\.{3}\\s*)?(?<variable>\\$.+?)(?:[\\s*]|$)/";
+        $regex = "/@{$this->tag->getName()}\\s+({$type}\\s*)?(&\\s*)?(\\.{3}\\s*)?(?<variable>\\$[a-zA-Z_]+[a-zA-Z0-9_]*)(?:.*|$)/";
```

Comment:
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.58.1/src/DocBlock/TypeExpression.php#L34 should be used instead